raiden-cli/package.json

```diff
@@ -31,6 +31,7 @@
     "typescript": "^4.2.3"
   },
   "dependencies": {
+    "@ethersproject/constants": "^5.0.10",
```

Comment:
If possible, I think it´d be good to not add the internal projects as dependencies for the user-facing projects; you can access `constants` from `ethers` import directly

Reply:
✔️